src/content/chainlink-automation/overview/getting-started.mdx

```diff
@@ -92,27 +92,27 @@ Click the tabs below to use Chainlink Automation with each type of trigger:
 
-      3. Click **Register new Upkeep** and select **log trigger**.
+      1. Click **Register new Upkeep** and select **log trigger**.
 
-      4. Under _Contract to automate_, enter `0xcf39b55F3E583DdEc8D418dB4708A71aA8453a83`. This is a simple `iLogAutomation`-compatible example contract that will increment a counter when a log is detected. The code is [here](https://sepolia.arbiscan.io/address/0xe817e4A71C69C72C01B31906F9F8591FbaB6b448#code). Click **Next**.
+      1. Under _Contract to automate_, enter <Address contractUrl="https://sepolia.arbiscan.io/address/0xe817e4A71C69C72C01B31906F9F8591FbaB6b448" />. This is a simple `iLogAutomation`-compatible example contract that increments a counter when a log is detected. View the source code [here](https://sepolia.arbiscan.io/address/0xe817e4A71C69C72C01B31906F9F8591FbaB6b448#code). Click **Next**.
 
-      5. Under _Contract emitting logs_, enter `0xeB4ABE57c2ba0A467273f9b7fDed130AD863e923`. This is the contract Automation will listen to for emitted logs. The code is [here](https://sepolia.arbiscan.io/address/0x1260206b960bB07F12d48C19fad505CeFc071bDd#code). Click **Next**.
+      1. Under _Contract emitting logs_, enter <Address contractUrl="https://sepolia.arbiscan.io/address/0x1260206b960bB07F12d48C19fad505CeFc071bDd" />. This is the contract Automation will listen to for emitted logs. View the source code [here](https://sepolia.arbiscan.io/address/0x1260206b960bB07F12d48C19fad505CeFc071bDd#code). Click **Next**.
 
          <ClickToZoom src="/images/automation/getting-started/gs_ltu_log_trigger_details.png" />
 
-      6. Under _Emitted log_ select **Bump** from the dropdown menu. This is the log signature Automation will look for.
+      1. Under _Emitted log_ select **Bump** from the dropdown menu. This is the log signature Automation will look for.
 
-      7. _Log index topic filters_ are optional filters to narrow the logs you want to trigger your upkeep. For our example enter your public key address under _addr_ and leave the _num_ field empty. Later when you call the _bump_ function to emit the log, your `msg.Sender` address will be emitted in the log, triggering your upkeep. Click **Next**.
+      1. _Log index topic filters_ are optional filters to narrow the logs you want to trigger your upkeep. For this example, enter your public key address under _addr_ and leave the _num_ field empty. Later when you call the _bump_ function to emit the log, your `msg.Sender` address will be emitted in the log, triggering your upkeep. Click **Next**.
 
          <ClickToZoom src="/images/automation/getting-started/gs_ltu_emit_log_details.png" />
 
-      8. Enter an _Upkeep name_, your public key address under _Admin Address_, 500000 under _Gas limit_, and 0.1 under _Starting balance (LINK)_.
+      1. Enter an _Upkeep name_, your public key address under _Admin Address_, 500000 under _Gas limit_, and 0.1 under _Starting balance (LINK)_.
```

Comment:
Import `CopyText` then add copy/paste component like this:
`<CopyText text="500000" code/>`
`<CopyText text="0.1" code/>`